tox.ini

```
@@ -3,18 +3,24 @@ skipsdist = True
 usedevelop = True
 minversion = 1.8
 envlist =
-    flake8-py{27,37},
-    twine-check-py{27,37},
-    py{27,35,36}-dj111
+    flake8-py37,
```

Is this comma needed?
```suggestion
    flake8-py37
```